.changeset/witty-spies-sin.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+ensure saved prerendered assets are stored as file and not as a folder
```

Comment:
I haven't looked closely, but assuming this is a bug, we should use the `fix: ` prefix for consistency in the changelog https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md
```suggestion
fix: ensure saved prerendered assets are stored as file and not as a folder
```